metrics-extensions/modules/grafana-dashboards/dashboards/ballerina-metrics-grafana-dashboard-prometheus.json

```diff
@@ -1424,49 +1589,49 @@
       "steppedLine": false,
       "targets": [
         {
-          "$$hashKey": "object:1460",
-          "expr": "rate({__name__=~\".*HttpClient_1XX_requests_total_value\", instance=~\"$instance\", action=~\"$http_client_action\"}[1m])",
+          "expr": "rate({__name__=~\"requests_total_value\", instance=~\"$instance\", action=~\"$http_client_action\",connector_name=\"ballerina/http/Client\",http_status_code=\"1xx\"}[1m])",
```

Comment:
Shouldn't the connector_name be "ballerina/http/HttpClient" ?
Shall we search for all occurrences of "ballerina/http/Client" and verify.